spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/config/AdminDefaultEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.admin.config;
+
+
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.config.YamlPropertiesFactoryBean;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.Ordered;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.core.env.MutablePropertySources;
+import org.springframework.core.env.PropertySource;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+
+import java.util.*;
+import java.util.function.Consumer;
+
+/**
+ * Contributes the values from {@code admin.yml} if it exists, before any of Spring Boot's normal
+ * configuration contributions apply. This has the effect of supplying overridable defaults
+ * to the various Spring Cloud Dataflow Admin SPI implementations
+ * that in turn override the defaults provided by Spring Boot.
+ *
+ * @author Josh Long
+ */
+public class AdminDefaultEnvironmentPostProcessor implements EnvironmentPostProcessor, Ordered {
+
+	protected Resource resource = new ClassPathResource("/admin.yml");
```

Comment:
Configurability should probably only apply to the path itself (not the `Resource`), but let's make this `private` for now and revisit that customizability when this moves to boot itself.